tests/baselines/reference/commentLeadingCloseBrace.js

```diff
@@ -0,0 +1,24 @@
+//// [commentLeadingCloseBrace.ts]
+declare function commentedParameters(...args): any;
+
+function ifelse() {
+    if (commentedParameters(1, 2)) {
+        /*comment1*/
+        commentedParameters(3, 4);
+        /*comment2*/
+    } else {
+        commentedParameters(5, 6);
+    }
+}
+
+//// [commentLeadingCloseBrace.js]
+function ifelse() {
+    if (commentedParameters(1, 2)) {
+        /*comment1*/
+        commentedParameters(3, 4);
+    /*comment2*/

```

Comment:
not sure I understand your comment. do you emit leading/trailing comments of each element in node-list